lua/plugins/editing/autocomplete.lua

```diff
@@ -141,8 +141,22 @@ return {
       opts.experimental = {
         ghost_text = false,
       }
-
       return opts
     end,
   },
+  {
```

Comment:
nit: maybe put this codeium stuff at the top since its the main thing in this file